cmd/tailscale/cli/risks.go

```diff
@@ -77,7 +77,7 @@ func presentRiskToUser(riskType, riskMessage, acceptedRisks string) error {
 	for left := riskAbortTimeSeconds; left > 0; left-- {
 		msg := fmt.Sprintf("\rContinuing in %d seconds...", left)
 		msgLen = len(msg)
-		printf(msg)
+		print(msg)
```

Comment:
This isn't equivalent. The printf before was our indirection to work in wasm/etc:

```go
var Stderr io.Writer = os.Stderr
var Stdout io.Writer = os.Stdout

func errf(format string, a ...any) {
	fmt.Fprintf(Stderr, format, a...)
}

func printf(format string, a ...any) {
	fmt.Fprintf(Stdout, format, a...)
}
```

By using `print`, you're using the implementation's `print` (not even part of the language!)

So I'd do `printf("%s", msg)` here

Reply:
oh snap... totally didn't notice this was a local function and not a builtin.  fixed.

Reply:
I also didn't realized that `print` and `println` weren't part of the language.  TIL  https://go.dev/ref/spec#Bootstrapping